### DISCUSSION TITLE
Importer doesn't recognize kimai export

### DISCUSSION THREAD

**Discussion by maurice-kpw** [kimai/kimai#4434] [Plugins]
Good morning :) 

We want to move kimai from our webhosting to our own server.
Both installations are kimai 2.2.1.

I'm exporting the whole time range for a specific user with the export in the old installation.
Using the Importer Plugin (2.5.1) I want to import this csv-file kimai created.

The importer throws the following error: "Invalid file given, missing and/or invalid columns: Rate, HourlyRate, InternalRate, FixedRate"
This error also came up in the previous version of the importer.
This error also gets thrown when using an export from the "All times"-page

Am I missing something here? Why doesn't kimai produce a file kimai can read?

I appreciate the help! 

Kind regards,
Maurice

**Comment by kevinpapst** [comment]
Becuase this is not the right way to move an installation. 
RTM: https://www.kimai.org/documentation/backups.html#create-a-database-backup
